common/djangoapps/course_category/admin.py

```diff
@@ -7,6 +7,36 @@
 from models import CourseCategory, Program
 
 
+class ProductModelMultipleChoiceField(forms.ModelMultipleChoiceField):
+    '''
+    Redefinition of clean method.
+
+    Values that original method recieves by m2m from CourseOverview are course_id (str),
+    while CourseLocator needed.
+    '''
+    def clean(self, value):
+
+        value = [CourseKey.from_string(v) for v in value]
+        return super(ProductModelMultipleChoiceField, self).clean(value)
+
+
+class ProgramForm(forms.ModelForm):
+
+    products = ProductModelMultipleChoiceField(
+        queryset=CourseOverview.objects.filter(product=True),
+        required=False,
+        label='Shippable or downloadable products'
+    )
```

Comment:
Renaming field was done with 'label' nor with Meta class. Do You have some objections?

Reply:
Nope. It's ok.